p2p/discv5/node_test.go

```diff
@@ -73,7 +73,7 @@ var parseNodeTests = []struct {
 	{
 		rawurl: "enode://1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439@127.0.0.1:foo",
 		// net/url.Parse(rawurl) returns an error with rawurl and why the parse failed.
-		wantError: `parse enode://1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439@127.0.0.1:foo: invalid port ":foo" after host`,
+		wantError: `invalid port`,
```

Comment:
Why did this file change?

Reply:
The output of this test depends on what version of go is being run. It was modified in https://github.com/celo-org/celo-blockchain/commit/f960cf71d8 to work for the version of Go used in Circle, but was failing for me locally with Go 1.11.5. I would like to make it work more universally so that I can run go test locally.